docs/reference/query-dsl/text-expansion-query.asciidoc

```diff
@@ -76,4 +76,31 @@ GET my-index/_search
    }
 }
 ----
-// TEST[skip: TBD]
\ No newline at end of file
+// TEST[skip: TBD]
+
+[discrete]
+[[optimizing-text-expansion]]
+=== Optimizing the search performance of the text_expansion query
+
+https://www.elastic.co/blog/faster-retrieval-of-top-hits-in-elasticsearch-with-block-max-wand[Max WAND] 
+optimization technique used by {es} to skip over documents that don't score 
+competitively against the current best matching documents. However, the weighted 
+terms generated by the ELSER model don't work well with the Max WAND 
```

Comment:
The rest of the doc uses the phrase `token-weight pairs` rather than `weighted terms`, to be consistent `tokens` should be preferred to `terms`.

This could be rewritten as "However, the tokens generated by the ELSER...."